### PR TITLE
show valid default asset on send page

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

File: src/pages/wallet/SendPage.tsx [hunk 1]
```diff
@@ -343,12 +348,7 @@ const SendPage = () => {
     }
   }, [chain, trigger, recipient])
 
-  const filteredAssets = useMemo(
-    () => availableAssets.filter(({ symbol }) => !symbol.endsWith("...")),
-    [availableAssets]
-  )
-
-  const assetsByDenom = filteredAssets.reduce(
+  const assetsByDenom = availableAssets.reduce(
```
Comment (ericHgorski) [inline src/pages/wallet/SendPage.tsx]:
```suggestion
  const assetsByDenom = filteredAssets.reduce(
```

**Review by restorenode — CHANGES_REQUESTED** [review]
After carrying out a transaction, the asset field will then update based on the filtered assets.

Do we want the Asset field to update after a user commits a transaction?

For example, if you send some LUNA to a wallet, after the transaction is successful, the asset field will update based on the filteredAssets variable to something like axlUSDC.

**Comment by ericHgorski** [comment]
> After carrying out a transaction, the asset field will then update based on the filtered assets.
> 
> Do we want the Asset field to update after a user commits a transaction?
> 
> For example, if you send some LUNA to a wallet, after the transaction is successful, the asset field will update based on the filteredAssets variable to something like axlUSDC.

Fixed. We should be able to pass those values directly to the `reset` call but it's not working for some reason...

https://react-hook-form.com/docs/useform/reset

File: src/pages/wallet/SendPage.tsx (hunk 1 above)
Comment (restorenode) [reply src/pages/wallet/SendPage.tsx]: @ericHgorski what was the reason for changing this line?

**Comment by ericHgorski** [comment]
> After carrying out a transaction, the asset field will then update based on the filtered assets.
> 
> Do we want the Asset field to update after a user commits a transaction?
> 
> For example, if you send some LUNA to a wallet, after the transaction is successful, the asset field will update based on the filteredAssets variable to something like axlUSDC.

Is this resolved now on your end?

**Review by restorenode — APPROVED** [review]

**Pull request merged**